pkg/private/tar/tar_writer.py

```diff
@@ -49,7 +49,8 @@ def __init__(self,
                create_parents=False,
                allow_dups_from_deps=True,
                default_mtime=None,
-               preserve_tar_mtimes=True):
+               preserve_tar_mtimes=True,
+               compresslevel=None):
```

Comment:
To keep things consistent, can you rename to `compress_level`?

Reply:
The current word `compresslevel` is from `gzip.GzipFile(compresslevel=...)`. So how do you think about the two names `compresslevel` and `compress_level`?

Reply:
Per [the Starlark style guide](https://bazel.build/rules/bzl-style#naming-convention), the preferred name should be `compress_level`. I understand that there may be an existing API that does not conform. However, I suggest that does not mean that this non-conforming API dictate what we implement today.

@aiuto Any thoughts?